src/map/anim/Map.ZoomAnimation.js

```diff
@@ -104,6 +104,14 @@ L.Map.include(!zoomAnimated ? {} : {
 			zoom: zoom,
 			noUpdate: noUpdate
 		});
+
+
+		// This shouldn't be needed if browsers fired ontransitionend consistently
+		setTimeout(L.bind(function(){
```

Comment:
It does in travis:
![2015-10-06-111923_699x60_scrot](https://cloud.githubusercontent.com/assets/1470389/10304717/28669c6a-6c1c-11e5-91a1-b0b9e8ebd51b.png)